src/db/plugins/simple/SimpleDatabasePlugin.hxx

```diff
@@ -86,6 +86,10 @@ public:
 		return *root;
 	}
 
+	const AllocatedPath GetCachePath() noexcept {
+		return cache_path;
+	}
```

Comment:
This allocates a copy of the path; but you don't need that here; all you want is `bool HasCache() const noexcept`.

Reply:
fixed